src/services/applications.js

```diff
@@ -1,11 +1,33 @@
 const { Applications } = require('../database')
+const {
+  ValidationError,
+  NotFoundError,
+} = require('../errors')
 
 async function create (payload) {
+  const { app_id } = payload
+
+  const existingApplication = await Applications.get(app_id).catch(() => null)
+
+  if (existingApplication) {
+    throw new ValidationError(['Application already exists'])
+  }
+
   return Applications.create(payload)
 }
 
-async function show () {
-  return { foo: 'ao', bar: true }
+async function show (id) {
+  try {
+    const application = await Applications.get(id)
+
+    return application
+  } catch (error) {
+    if (error.name === 'ValidationException') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
    if (error.name === "ValidationException") {
```